star-wars.Rmd

```diff
@@ -5,16 +5,32 @@ output: github_document
 
 ```{r setup, include = FALSE}
 knitr::opts_chunk$set(echo = FALSE)
+library(dplyr)
+library(tidyr)
+library(knitr)
 ```
 
 ## Most accomplished
 
 Below we look at which 5 characters from Star Wars are the most accomplished pilot (flew the fewest number of starships):
 
 ```{r table-of-most-accomplished-pilots}
+accomp_pilots <- starwars %>% 
+  unnest(films) %>% 
+  group_by(name) %>% 
+  tally() %>% 
+  arrange(desc(n)) %>% 
+  head(10)
```

Comment:
wrong dataset used for `unnest` function. should be `starships` instead of `film`.

```suggestion
accomp_pilots <- starwars %>% 
  unnest(starships) %>% 
  group_by(name) %>% 
  tally() %>% 
  arrange(desc(n)) %>% 
  head(10)
```